pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -570,6 +570,12 @@ async def create_pulp3_content(self):
             treeinfo_serialized["repositories"] = {'.': pulp2_repo.pulp3_repository.pk}
             # Pulp 2 did not support addon repositories, so we should not list them here either
             treeinfo_serialized['addons'] = []
+            # Pulp 2 only supported variants that are in the root of the repository
+            variants = []
+            for variant in treeinfo_serialized['variants']:
+                if variant['repository'] == '.':
```

Comment:
Can there be more than one? I think it's a path to an RPM repo, so there can only be one in '.'. What do you think?

Reply:
I was not sure if there can be multiple variants that point to the same repo.